packages/blockchain-wallet-v4-frontend/src/modals/BuySell/QuoteCountDown/QuoteCountDown.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { DefaultTheme, withTheme } from 'styled-components'
+
+import CircularProgressBar from 'components/CircularProgressBar'
+
+import { useCountDown } from '../hooks/useCountDown'
+import { Highlight, ProgressWrapper, Root, Text } from './QuoteCountDown.model'
+
+type Props = {
+  date: Date
+  theme: DefaultTheme
+  totalMs: number
+}
+
+export const QuoteCountDown = withTheme(({ date, theme, totalMs }: Props) => {
+  const { isCompleted, isCompletingSoon, percentage, timer } = useCountDown(date, totalMs)
+
+  return (
+    <Root>
+      <ProgressWrapper>
+        <CircularProgressBar
+          strokeWidth={15}
+          percentage={percentage}
+          styles={{
+            strokeLinecap: 'butt',
+            trailColor: theme.blue100
+          }}
+        />
+      </ProgressWrapper>
+
+      <Text>
+        <FormattedMessage id='modals.simplebuy.quote.countdown' defaultMessage='New quote in:' />{' '}
+        <Highlight isHighlighted={isCompletingSoon}>
+          {isCompleted ? (
+            <FormattedMessage id='modals.simplebuy.quote.soon' defaultMessage='soon' />
+          ) : (
+            timer
+          )}
+        </Highlight>
+      </Text>
+    </Root>
+  )
+})
```

Comment:
Personally, I switched to using named exports long time ago since they work much better with auto imports in IDE. In turn, that enforces consistency of import names across the codebase. Happy to discuss and change the PR if needed.